chatterbot/storage/mongodb.py

```diff
@@ -227,6 +228,9 @@ def get_response_statements(self):
         _response_query = {
             'in_response_to': {
                 '$ne': None
+            },
+            'persona': {
+                '$not': re.compile('^bot:*')
```

Comment:
I'm quite impressed that it's possible to pass a Python regular expression to Mongo DB. I'd be interested to know how this works on the database side of things.